platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/internal/reconnect/ReconnectNodeRemover.java

```diff
@@ -310,8 +329,9 @@ private void handleLeaf(final long path, final ReplacedNodeType replacedType, fi
         // If we think we need to remove a key but find out that the leaf has actually just been moved,
         // then we don't actually need to remove it.
         keysToBeRemoved.remove(key);
-
+        //logger.info(RECONNECT.getMarker(), "Adding to key to {}", encounteredKeys.getClass().getName());
         encounteredKeys.add(key);
+        //logger.info(RECONNECT.getMarker(), "handled leaf at path {}", path);
```

Comment:
You added a lot of commented code. Why not just removing it. Or for loggers: Use lowest log level.